src/main/java/com/epam/asset/tracking/api/UserController.java

```diff
@@ -102,4 +115,24 @@ public BusinessProviderDTO getUserData (
 
 		return userData;
 	}
+
+    @GetMapping(value = "retrievePassword/{username}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Sending a new random password to the user")
+    @ApiResponses({ @ApiResponse(code = 200, message = "Send an email to the user with the new password", response = BusinessProvider.class),
+            @ApiResponse(code = 405, message = "Invalid Username provided") })
+    @ResponseStatus(HttpStatus.OK)
+    public void sendUserPassword (
+            @ApiParam(value = "Sends a random password to the user email, if the email is valid", required = true) @PathVariable @Valid String username,
```

Comment:
@Valid annotations is only useful when validating nested properties in on POJOs. What were you trying to validate?